css/first.css

```diff
@@ -0,0 +1,11 @@
+.header{
```

Comment:
Please add a space between the element and the character {. So the linter don't mark it as an error.